app/assets/stylesheets/application.css

```diff
@@ -16,3 +16,37 @@
  textarea {
    resize: none;
  }
+
+.index {
+
+   border-bottom: solid;
+   border-bottom-color: grey;
+ }
+
+/* #edit {
+  background-color: #4CAF50;
+}
+
+#delete {
+  background-color: #f44336;
+} */
+
+#text {
+  color: white !important;
```

Comment:
what is !important?

Reply:
the !important overrides all other css. We were trying to make our edit and delete buttons not look like hyperlinks but haven't figured it out yet.